src/main/java/teammates/logic/api/TaskQueuer.java

```diff
@@ -0,0 +1,277 @@
+        
+        addTask(TaskQueue.FEEDBACK_SESSION_PUBLISHED_EMAIL_QUEUE_NAME,
+                TaskQueue.FEEDBACK_SESSION_PUBLISHED_EMAIL_WORKER_URL, paramMap);
+    }
+    
+    /**
+     * Schedules for feedback session unpublished email to be sent.
+     * 
+     * @param courseId the course ID of the feedback session
+     * @param feedbackSessionName the name of the feedback session
+     */
+    public void scheduleFeedbackSessionUnpublishedEmail(String courseId, String feedbackSessionName) {
+        Map<String, String> paramMap = new HashMap<String, String>();
+        paramMap.put(ParamsNames.EMAIL_COURSE, courseId);
+        paramMap.put(ParamsNames.EMAIL_FEEDBACK, feedbackSessionName);
+        
+        addTask(TaskQueue.FEEDBACK_SESSION_UNPUBLISHED_EMAIL_QUEUE_NAME,
+                TaskQueue.FEEDBACK_SESSION_UNPUBLISHED_EMAIL_WORKER_URL, paramMap);
+    }
+    
+    /**
+     * Schedules for course registration to be sent to the specified student.
+     * 
+     * @param courseId the target course ID
+     * @param studentEmail the email address of the student
+     */
+    public void scheduleCourseRegistrationInviteToStudent(String courseId, String studentEmail) {
+        Map<String, String> paramMap = new HashMap<String, String>();
+        paramMap.put(ParamsNames.COURSE_ID, courseId);
+        paramMap.put(ParamsNames.STUDENT_EMAIL, studentEmail);
+        
+        addTask(TaskQueue.COURSE_JOIN_REMIND_EMAIL_QUEUE_NAME, TaskQueue.COURSE_JOIN_REMIND_EMAIL_WORKER_URL, paramMap);
+    }
+    
+    /**
+     * Schedules adjustments to be done to responses of a feedback session in the database
+     * after change is done to a course, typically after enrollment of new students
+     * or re-enrollment of old students, or both.
+     * 
+     * @param courseId the course ID of the feedback session
+     * @param feedbackSessionName the name of the feedback session
+     * @param enrollmentList the list of enrollment details
+     */
+    public void scheduleFeedbackResponseAdjustmentForCourse(String courseId, String feedbackSessionName,
+                                                            List<StudentEnrollDetails> enrollmentList) {
+        Map<String, String> paramMap = new HashMap<String, String>();
+        paramMap.put(ParamsNames.COURSE_ID, courseId);
+        paramMap.put(ParamsNames.FEEDBACK_SESSION_NAME, feedbackSessionName);
+        
+        String enrollmentDetails = JsonUtils.toJson(enrollmentList);
+        paramMap.put(ParamsNames.ENROLLMENT_DETAILS, enrollmentDetails);
+        
+        addTask(TaskQueue.FEEDBACK_RESPONSE_ADJUSTMENT_QUEUE_NAME,
+                TaskQueue.FEEDBACK_RESPONSE_ADJUSTMENT_WORKER_URL, paramMap);
+    }
+    
+    /**
+     * Schedules for the given list of emails to be sent.
+     * 
+     * @param emails the list of emails to be sent
+     */
+    public void scheduleEmailsForSending(List<EmailWrapper> emails) {
```

Comment:
It is not. It is the equivalent of `EmailSender#sendEmails` as evident from its call sites.